### DISCUSSION TITLE
Webradio Bookmarks

### DISCUSSION THREAD

**Discussion by R101G** [rern/rAudio#1129] [Feature Requests]
@rern 
This has been mentioned before, but I would like to make a formal request for this feature to be added when/if you have the time.

**Comment by ceejayemm** [comment]
I would support this request too please.

Thanks

Chris

**Comment by rern** [comment]
@R101G,
Chris,

Please try the UPDATE branch.

**Comment by ceejayemm** [comment]
@rern 
Thanks very much, this seems to work exactly how I wanted it to.  

The only problem, as previously noted in #1079, is that when using (rAudio-1 provided) SharedData, bookmarks can only be added/removed via the 'server' machine.  I asked there if this was by design ?  If so its not a problem to me I just wanted to clarify that this was what you intended.

Many thanks for very quick response.

Chris

**Reply by rern** [reply]
Hi Chris,

Try: `chown -h mpd:audio $dirplaylists`

**Reply by ceejayemm** [reply]
@rern 
I can't really try this until #1144 is fixed. I guess.  At the moment it returns:

rAudio-1:~  #  chown -h mpd:audio $dirplaylists
chown: missing operand after 'mpd:audio'
Try 'chown --help' for more information.

as I guess $dirplaylist variable is not known ?

Chris

**Comment by R101G** [comment]
@rern 
Thanks for this. There are a couple of things :)

The old problem of the metadata missing until the next refresh reappears if I use this for RP stations.

I think it would look better if the default star icon was an option, rather than the station logo. To me, the different icon styles make the library page look untidy. (Would it be possible to add the system colour to the webradio icon too?)

I think if the one-touch replace and play function is selected, it should be observed here, otherwise there is no reduction in click-count.

**Reply by ceejayemm** [reply]
I get album covers as well web radio logos when saved as bookmarks, I don't see any stars at any time.  I prefer it the way it is.

Can't comment on the RP stations issues as I don't use these stations.

Chris

**Comment by AndyFx12** [comment]
@rern 
Thanks for this small but very fine feature!
